### ISSUE TITLE
Deprecation warning

### ISSUE THREAD

**Issue by damir** [schrockwell/live_inspect#12]
Hi Rockwell, 

LiveView 0.20.0 prints this warning:

```
warning: LiveInspect.Inspector.preload/1 is deprecated, use update_many/1 instead
```
